src/utils/execa.js

```diff
@@ -0,0 +1,9 @@
+const { env } = require('process')
+
+// This is a thin layer on top of `execa` that allows consumers to provide an
+// alternative path to the module location, making it easier to mock its logic
+// in tests (see `tests/utils/mock-execa.js`).
+// eslint-disable-next-line import/no-dynamic-require
+const execa = require(env.NETLIFY_CLI_EXECA_PATH || 'execa')
```

Comment:
To keep static imports (which might be useful when switching to ES modules in the future), should we do something more like this?

```
const execaLib = require('execa')
const execa = env.NETLIFY_CLI_EXECA_PATH === undefined ? execaLib : require(env.NETLIFY_CLI_EXECA_PATH)
```

Reply:
But we'll still have a dynamic import (when we're requiring the env variable), so the change you're proposing just means having one instead of two. Do you still think it's worth doing?

Reply:
Yes, exactly, that's what I meant: one instead of two.
It's not super important, but I am wondering for example if type inference on `execa` (once we start using TypeScript) might be better that way? Regardless, this is very nit-picky, so please feel free to discard :+1:

Reply:
I'm happy either way. Done in https://github.com/netlify/cli/pull/3047/commits/bec7f55ace9a61823d9e7decb9584b5df0c22873.